metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Application cookbook which installs and configures Consul.'
 long_description 'Application cookbook which installs and configures Consul.'
-version '3.1.0'
+version '3.1.1'
```

Comment:
Would you please drop this change from the PR? Version number changes occur on master as part of the release process.